iotfunctions/__init__.py

```diff
@@ -1,3 +1,8 @@
+# Licensed Materials - Property of IBM
+# 5737-M66, 5900-AAA, 5900-A0N, 5725-S86, 5737-I75
+# (C) Copyright IBM Corp. 2020 All Rights Reserved.
```

Comment:
I believe you have to add 2021 in the copyright line because the file was last updated in 2021
```
# (C) Copyright IBM Corp. 2020, 2021 All Rights Reserved.
```